### PR TITLE
document using robot.messageRoom

### PR DESCRIPTION
**Pull request by twellspring**
Adding documentation for robot.messageRoom per #1085 

Note:  I was only able to test this using the hubot-slack integration

### CODE REVIEW

File: docs/scripting.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+Â---
```
Comment (pchaigno) [inline docs/scripting.md]: Looks like some random character manage to find its way here ;)

**Comment by twellspring** [comment]
Anything else needed to get this merged?

**Comment by technicalpickles** [comment]
@twellspring as @pchaigno there's a random character in the YAML front matter, which would mess up rendering this on hubot.github.com.

**Comment by twellspring** [comment]
@technicalpickles anything else needed to get this merged?

File: docs/scripting.md [hunk 2]
```diff
@@ -75,6 +74,33 @@ The `robot.hear /badgers/` callback sends a message exactly as specified regardl
 
 If a user Dave says "HAL: open the pod bay doors", `robot.respond /open the pod bay doors/i` callback sends a message "Dave: I'm afraid I can't let you do that."
 
+## Messages to a room or user
+
+Messages can be sent to a specified room or user using the messageRoom function.
+For slack rooms, do not put the `#` prefix in front of the room name.
```
Comment (bhuga) [inline docs/scripting.md]: This is adapter-specific; let's keep that out of this document.

**Comment by bhuga** [comment]
> anything else needed to get this merged?

Sorry, the front matter is still not quite right. It needs three `---` on a line to start. If you check out `adapters.md` on github you'll see that the front matter parses into a little table; if you see that table, we're good to go.

**Comment by twellspring** [comment]
took care of the ---  and removed the Slack specific line

**Comment by bkeepers** [comment]
Thanks, and sorry for the delay.

**Pull request merged**